contracts/dev-utils/contracts/src/LibAssetData.sol

```diff
@@ -115,6 +120,21 @@ contract LibAssetData is
                     balance = scaledBalance;
                 }
             }
+        } else if (assetProxyId == STATIC_CALL_PROXY_ID) {
+            // Encode data for `staticCallProxy.transferFrom(assetData,...)`
+            bytes memory transferFromData = abi.encodeWithSelector(
+                _ASSET_PROXY_TRANSFER_FROM_SELECTOR,
+                assetData,
+                address(0),  // `from` address is not used
+                address(0),  // `to` address is not used
+                0            // `amount` is not used
+            );
+
+            // Check if staticcall would be successful
+            (bool success,) = _STATIC_CALL_PROXY_ADDRESS.staticcall(transferFromData);
+
+            // Success means that the staticcall can be made an unlimited amount of times
+            balance = success ? _MAX_UINT256 : 0;
```

Comment:
That call can be made an unlimited amount of times at the particular block in which this gets called (not accounting for gas). This should always be true since the staticcall can't update state and will therefore never have side effects that would prevent the staticcall from failing if called again.